### PR TITLE
Document how to use TensorBoardLogger with fsspec

### PR DESCRIPTION
**Pull request by leoleoasd**
## What does this PR do?

Document how to use TensorBoardLogger with fsspec.

see https://github.com/Lightning-AI/lightning/pull/15728#issuecomment-1377304129

### Does your PR introduce any breaking changes? If yes, please list them.
None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by leoleoasd** [comment]
Do other loggers support fsspec paths? I'm currently only putting TensorBoardLogger in the document

File: docs/source-pytorch/common/remote_fs.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Working with different filesystems can be accomplished by appending a protocol l
     trainer = Trainer(default_root_dir="s3://my_bucket/data/")
     trainer.fit(model)
 
-You could pass custom paths to loggers for logging data.
+As for ``TensorBoardLogger``, by default, we use ``tensorboardX``. To use it with ``fsspec`` support, make sure you have ``tensorboard`` installed AND NO ``tensorflow`` installed in the current environment, as ``tensorboard`` will use ``tensorflow``'s GFile when detected, and it only uses fsspec on absence of ``tensorflow``.
```
Comment (awaelchli) [inline docs/source-pytorch/common/remote_fs.rst:20]: I'm ok with this specific explanation. 

Here is an alternative suggestion that might work better:
1. Here, explain in general terms: "For logging, remote filesystem support depends on the particular logger integration being used. Consult the documentation of the individual logger for more details (and here include a link to our logger page overview)."
2. In each logger, we declare whether it supports remote filesystems or not. For example, in TensorBoardLogger we would write: "This logger supports writing to remote fileystems (s3, gcs, etc.) if you have the tensorboard package installed". In others we write: "This logger does not support writing to remote FS".

Comment (leoleoasd) [reply docs/source-pytorch/common/remote_fs.rst:20]: Yes, this way is better. Tensorboard supports remote filesystems, for sure, and I think I can add support for CSVLogger. However, I'm not sure if other loggers, for example, mlflow and wandb, support remote filesystems.

**Comment by leoleoasd** [comment]
Blocked by #16158. Will update later...

**Comment by awaelchli** [comment]
Feel free edit the files and push it without local build. If there are errors, we can help resolve them. The build runs here in the CI.

**Review by awaelchli — APPROVED** [review]

**Comment by leoleoasd** [comment]
Let's wait until I got some time to add fsspec support for CSVLogger.

**Review by justusschock — APPROVED** [review]

**Comment by awaelchli** [comment]
@leoleoasd That's ok if you want to wait. From our side, it would also be fine to land this doc update and do the CSV support  in a separate PR. As you wish.

**Comment by awaelchli** [comment]
@leoleoasd Let's merge this PR? Is it ready?

**Comment by leoleoasd** [comment]
I'll update documentation for tensorbosrdlogger and csvlogger today and then it will be ready.

**Review by Borda — APPROVED** [review]
